packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/ApplicationLauncherDemo/ApplicationLauncherDemo.tsx

```diff
@@ -18,7 +18,8 @@ export class ApplicationLauncherDemo extends React.Component<null, ApplicationLa
         isOpen
       });
     };
-    this.onSelect = event => {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    this.onSelect = _event => {
```

Comment:
Curious here, if the '_event' var is not being used is there a reason we would not just use  the '() =>' notation here instead of applying the ignore to the linting rule? I could be missing why we are doing this so its more a question.  Happens more than once in this file. I can update each one I find once we determine if we want to leave it as is or make a change.

Reply:
Thanks for the call-out - these were updated per [Dan's comments](https://github.com/patternfly/patternfly-react/pull/3718#discussion_r378345721) above

Reply:
The underscore just eliminates the linter warning. In the case of a demo, it's desirable to show the expected variables.